### DISCUSSION TITLE
my tractor can't find the harvester to unload

### DISCUSSION THREAD

**Discussion by vitorhn19** [Courseplay/Courseplay_FS22#3068] [Q&A]
my tractor can't find the harvester to unload
help me! 
I can't play

**Comment by Tensuko** [comment]
Do you think some more Informations could be useful to help you?
If not, then I close this, if yes, then provide more informations pls.
Also: Read the ingame help menu!

**Reply by vitorhn19** [reply]
yes, i'am brazilian, i have dificulty for explicate, but I'm trying to get the tractor to load into the harvester and unload into the truck, but the tractor won't go to the harvester as it doesn't identify it, I'm going to have to stop playing if I don't fix it

**Reply by Tensuko** [reply]
Latest LS Version and CP Version?
What Combine?
Did you select the field and so on.
Screenshots also can help.

**Reply by vitorhn19** [reply]
updated game, updated cp for modhub 
X9 combine and jcb tractor
field selected
all right
but the tractor won't go to the harvester as it doesn't identify it

**Reply by Tensuko** [reply]
What X9 Combine.....
And what do you expect, is the combine full already?

**Reply by vitorhn19** [reply]
harvester X9 JOHN DEERE i mean
yes, it fills up and stops

**Reply by Tensuko** [reply]
Can you give me Screenshots or Video of how you do this?
Becouse I don't have any problem with unloading.
